lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py

```diff
@@ -453,7 +513,9 @@ def __init__(self):
             overprovision=dict(type='bool', default=True),
             single_placement_group=dict(type='bool', default=True),
             zones=dict(type='list'),
-            custom_data=dict(type='str')
+            custom_data=dict(type='str'),
+            plan=dict(type='dict'),
+            accept_terms=dict(type='bool', default=False)
```

Comment:
```suggestion
            accept_terms=dict(type='bool', default=False),
```
Personally I'd add a trailing comma here, so the next PR won't have to touch this line again.

Reply:
ok, this will be in my next push